src/pudl/metadata/fields.py

```diff
@@ -1302,7 +1306,64 @@
     },
     "ferc_qualifying_facility": {
         "type": "boolean",
-        "description": "Indicatates whether or not a generator is a qualifying FERC cogeneation facility.",
+        "description": "Indicatates whether or not a generator is a qualifying FERC cogeneration facility.",
+    },
+    "fgd_control_flag": {
+        "type": "boolean",
+        "description": "Indicatates whether or not a plant has a FGD control unit.",
```

Comment:
I think we should spell out "flue gas desulfurization" in the long descriptions for clarity, since it's used in an abbreviated form in so many shorter names and might be confusing to someone unfamiliar with emissions controls.